docs/meetings/2020/TechArea20200824.md

```diff
@@ -38,17 +39,40 @@
 -   Tech team priorities this week: <https://opensciencegrid.atlassian.net/secure/RapidBoard.jspa?rapidView=19&quickFilter=66>
 
 
+-   AI (Diego): Continue to build and scitokens packages for EL8.  Contact Jason Patton for software help.
+-   AI (Edgar): Build scitokens-issuer for EL8; Derek to merge pull request and make a new release.
+-   AI (Edgar): Build osg-gridftp for EL8 since osg-xrootd-standalone is blocked by osg-configure.
+
+
 ### Discussion
 
-None this week  
+None this week.
+
```

Comment:
```suggestion
-   AI (Diego): Continue to build and scitokens packages for EL8.  Contact Jason Patton for software help.
-   AI (Edgar): Build scitokens-issuer for EL8; Derek to merge pull request and make a new release.
-   AI (Edgar): Build osg-gridftp for EL8 since osg-xrootd-standalone is blocked by osg-configure.
```

Throwing plate review into the discussion

Reply:
What goes into the Software Team section, then?

Reply:
I tend to use it as the starting point for stuff that I want to cover during the meeting (COManage registration, plates, probably should've had the doc focus date in there, too) and discussion contains anything that comes about from those talking points.